### PR TITLE
synthetics-minion/kmullaney

### PR DESCRIPTION
**Pull request by keegoid-nr**
#### Checklist

- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)

### CODE REVIEW

**Comment by keegoid-nr** [comment]
I'm having trouble getting the minion to be scheduled to run on a node and pass the Lint and Test Charts check. The node says it is not ready.

**Comment by paologallinaharbur** [comment]
It is a timeout due to an old version of the kindCluster action, we are about to merge a fix for that.

The following in the workflow should fix the issue
```
uses: helm/kind-action@v1.2.0
```

**Comment by paologallinaharbur** [comment]
I updated your branch to make use of the updated action, now it is not failing 👍

File: charts/synthetics-minion/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: synthetics-minion
 description: New Relic Synthetics Containerized Private Minion (CPM)
 type: application
-version: 1.0.34
+version: 1.0.35
```
Comment (cleon-nr) [inline charts/synthetics-minion/Chart.yaml:5]: We'll need to bump this to `1.0.36`.

**Review by cleon-nr — APPROVED** [review]
@keegoid-nr

Thanks for these changes. They look good to me!

Since some of these updates are related to the top level `README`, this PR will also require approval from the owner of that directory as well.

**Review by roobre — APPROVED** [review]
Changes to `CONTRIBUTING.md` lgtm :)

File: charts/synthetics-minion/Chart.yaml (hunk 1 above)
Comment (cleon-nr) [reply charts/synthetics-minion/Chart.yaml:5]: Strangely it's showing that the last version is `1.0.34` from this diff, but we already have `1.0.35` merged in. Perhaps your fork just needs to be synced up again.

Comment (keegoid-nr) [reply charts/synthetics-minion/Chart.yaml]: bumped version

Comment (keegoid-nr) [reply charts/synthetics-minion/Chart.yaml]: the versions keep changing faster than this pr can get merged

**Pull request merged**